packaging/requirements/requirements-azure.txt

```diff
@@ -1,7 +1,7 @@
 packaging
 requests[security]
 azure-mgmt-compute>=2.0.0,<3
-azure-mgmt-network>=1.3.0,<2
+azure-mgmt-network==1.6.0
```

Comment:
It's unnecessary, switching to keyword syntax makes it work with old and new versions. The issue you're fixing is a positional issue only.